creme/billing/models/line.py

```diff
@@ -77,7 +81,7 @@ class Meta:
         app_label = 'billing'
         verbose_name = _('Line')
         verbose_name_plural = _('Lines')
-        ordering = ('created',)
+        ordering = ('order', 'created')
```

Comment:
I am not sure that this ordering is relevant in list-views :thinking: (& it's probably slower).
Maybe we should just use an explicit order when retrieving the lines related to a billing entity. But maybe it won't be consistent in different places (reports?)  :thinking: 
I propose we discuss that topic with the team.

Reply:
Hum... Good point